pom.xml

```diff
@@ -28,11 +28,52 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <java.version>1.7</java.version>
     <tomcat.version>7.0.52</tomcat.version>
-    <spring.version>4.3.8.RELEASE</spring.version>
+    <spring.version>5.1.7.RELEASE</spring.version>
```

Comment:
did we quadruple check that this spring version matches the cdd version ?

Reply:
just quintuple checked!

Reply:
lol